### ISSUE TITLE
Pegging one CPU

### ISSUE THREAD

**Issue by markriggins** [jwilder/dockerize#32]
When I run the -stdout on OSX virtualbox, it pegs one cpu.

**Comment by wilsonmar** [comment]
It would help if a performance benchmark study would measure the performance impact of using various Dockerize features, such as templating.

**Comment by atopuzov** [comment]
Not sure if mine is related so I opened a new issue https://github.com/jwilder/dockerize/issues/81

**Comment by basz** [comment]
when I have this

```
        "-stdout",  "/var/log/nginx/access.log", 
        "-stderr", "/var/log/nginx/error.log", 
```

two cpu are running 100%

removing either one frees one cpu.
remove both and things are normal.

This is on my mac (local), but also on an ubuntu production machine...

**Comment by jak** [comment]
I had exactly the same issue, only affecting one of two project despite having the same base image and same `dockerize` command running.

The only difference I found was one project was removing the log file before dockerize ran. Changing this, I no longer experience this problem.

**Comment by stepankuzmin** [comment]
I have the same issue. Removing lines below helps.

```
"-stdout",  "/var/log/nginx/access.log", 
"-stderr", "/var/log/nginx/error.log", 
```

**Comment by ghostsquad** [comment]
@stepankuzmin is this fixed in the newest version of Dockerize? It looks like the `hpcloud/tail` has been updated in the latest version.

**Comment by jwilder** [comment]
Should be fixed via #92
